appcell-resources/css/applicationmarket.css

```diff
@@ -6,10 +6,6 @@
     margin-bottom: 0px;
 }
 
-.ellipsisText{
```

Comment:
There is another CSS already handling ellipsis text, therefore, it is okay to remove this CSS.